Dockerfile

```diff
@@ -1,26 +1,21 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including some of the settings below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
+# The playbook to be run is specified via the PLAYBOOK_FILE env var
+# Setting a default of openshift_facts.yml as it's an informative playbook
+# that can help test that everything is set properly (inventory, sshkeys)
+ENV PLAYBOOK_FILE=plabyooks/byo/openshift_facts.yml \
+    ANSIBLE_HOST_KEY_CHECKING=False \
+    OPTS="-v" \
+    INSTALL_OC=true
 
-LABEL Name="openshift3/installer"
-LABEL Vendor="Red Hat" License=GPLv2+
-LABEL Version="v3.1.1.901"
-LABEL Release="6"
-LABEL BZComponent="aos3-installation-docker"
-LABEL Architecture="x86_64"
-LABEL io.k8s.description="Ansible code and playbooks for installing Openshift Container Platform." \
-      io.k8s.display-name="Openshift Installer" \
-      io.openshift.tags="openshift,installer"
```

Comment:
Might be good to have relevant metadata for this image, maybe @tdawson can help with recommendations.